R/axis.R

```diff
@@ -71,79 +71,78 @@ axis.gsplot <- function(object, ..., n.minor=0, tcl.minor=0.15, reverse=NULL) {
   
   sides <- user_args[[fun.name]]$side
   user_args[[fun.name]]$side <- NULL
+  user_args[[fun.name]]$n.minor <- n.minor
+  user_args[[fun.name]]$tcl.minor <- tcl.minor
   
-  for(i in sides){
-    arguments1 <- list(arguments = append(list(side=i), user_args[[fun.name]]), 
-                       gs.config=list(n.minor=n.minor, tcl.minor=tcl.minor, 
-                                      reverse=reverse))
-    to.gsplot <- list(arguments1)
-    names(to.gsplot) <- fun.name
+  for(side in sides){
+    # append the side and give it defaults if it doesn't exist
+    object <- modify_side(object, args = list(), side=side) 
     
-    object <- append(object, to.gsplot)
```

Comment:
we don't have `add_to_side`, which is really what is happening here, but it isn't just adding it as a new call, it is replacing the default `axis` call that is placed in when you create a new side. I wasn't quite sure how we should do that.